server/rooms.ts

```diff
@@ -1352,11 +1383,13 @@ export class GlobalRoomState {
 	}
 	getRooms(user: User) {
 		const roomsData: {
-			official: ChatRoomTable[], pspl: ChatRoomTable[], chat: ChatRoomTable[], userCount: number, battleCount: number,
+			chat: ChatRoomTable[],
+			sectionTitles: {[k: string]: string},
+			userCount: number,
+			battleCount: number,
 		} = {
-			official: [],
-			pspl: [],
 			chat: [],
+			sectionTitles: RoomSections.sectionNames,
```

Comment:
Might it be better to just use the section title directly in the chat list, instead of needing to bundle an id:title map?

Reply:
I did the math and: yes, that would be better. Since it also makes it easier to parse, I strongly recommend going with that approach instead.

Reply:
How would i build the dropdown menu on the client then?

Reply:
You could send the title rather than the ID. Shouldn't be very different.

Reply:
I can also do it myself.